README.md

```diff
@@ -1,13 +1,18 @@
 # `babel-plugin-styled-components`
 
-This plugin adds support for server-side rendering, for minification of styles and gives you a nicer debugging experience when using `styled-components`.
+This plugin is a highly recommended supplement to the base styled-components library, offering some useful features:
+
+* consistently hashed component classNames between environments (a must for server-side rendering)
+* automatic library usage tweaks that allow for some bytes to be dropped from your bundle
+* better debugging through automatic annotation of your styled components based on their context in the file system, etc.
+* various types of minification for styles and the tagged template literals styled-components uses
 
 ## Quick start
 
 Install the plugin first:
 
 ```
-npm install --save-dev babel-plugin-styled-components
+npm i -D babel-plugin-styled-components
```

Comment:
Shorthands like this one may look a bit too magical for beginners. What is the drawback of keeping the long notation?

Reply:
No drawback, just more verbose

Reply:
That’s good for commands in the docs, isn’t it? :)

Reply:
Yeah I think it should go back to the original version to be clearer for beginners, it doesn't hurt :blush: